integration_tests/assets/docker-compose.wsgi-patch.override.yml

```diff
@@ -0,0 +1,9 @@
+services:
+  sync:
+    depends_on:
+      - wsgi-patch
+    environment:
+      TARGETS: ''
+    ulimits:
+      nproc: 65535
+      nofile: 100
```

Comment:
why put ulimits on the sync service?
Also, why set "TARGETS" to `''`? 
The `TARGETS` is used to specify which service you want to bring up, instead of `depends`(which only cares about container being up and not the actual service running in the container).

Reply:
`ulimits` was an artefact of something I tried, it's gone now. Regarding `Targets`, I simply relied on the dockerfile from the other tests in this repo, not to reinvent the wheel or risk to go against how `wazo-test-helpers` works.